Userland/Utilities/nc.cpp

```diff
@@ -36,13 +54,15 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     bool udp_mode = false;
     const char* target = nullptr;
     int port = 0;
+    int max_tcp_rcv_buff_size_input = -1;
```

Comment:
Please don't randomly abbreviate identifiers. We don't have a limit on horizontal length.